presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -310,6 +310,13 @@ Property Name                                      Description
                                                    can be used to use a different location for each user.
 
 ``hive.translate-hive-views``                      Enable translation for Hive views. (experimental)            ``false``
+
+``hive.legacy-hive-view-translation``              Use the legacy algorithm to translate Hive views.            ``false``
+                                                   Set this property to ``true`` if you encounter unhandled 
```

Comment:
I would drop the `Set this property to ``true`` if you encounter unhandled AST token exceptions with the new experimental algorithm.` part but do not feel strongly about it.